src/icons/SuccessSnackbar/SuccessSnackbar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getThemeColors } from '../../_helpers/theme';
+import colors from '../../tokens/colors';
+
+function Test({ width, height, fill }) {
```

Comment:
Fix this.

Reply:
it was by mistake. rectified immediately.